src/index.js

```diff
@@ -323,7 +323,15 @@ class Dropzone extends React.Component {
       type: 'file',
       style: { display: 'none' },
       multiple: supportMultiple && multiple,
-      ref: el => this.fileInputEl = el, // eslint-disable-line
+      ref: el => {
+        this.fileInputEl = el
+        this.fileInputEl.onclick = event => {
```

Comment:
I'm wondering if this can break some code that uses even handler on this ref. Also, this will create a memory leak when element is unmounted. Please use `addEventListener` and `removeEventListener` for this.